packages/api/endpoints/workflows.js

```diff
@@ -26,20 +30,28 @@ function list(event, cb) {
  * @param {callback} cb - aws lambda callback function
  * @returns {undefined} undefined
  */
-function get(event, cb) {
+async function get(event, cb) {
   const name = _get(event.pathParameters, 'name');
 
-  const key = `${process.env.stackName}/workflows/list.json`;
-  S3.get(process.env.bucket, key)
-    .then((file) => {
-      const workflows = JSON.parse(file.Body.toString());
+  const workflowKey = `${process.env.stackName}/workflows/list.json`;
+  try {
+    const { Body } = await aws.getS3Object(process.env.bucket, workflowKey);
```

Comment:
Since fetching the workflows list from S3 is done in both `list` and `get`, it should be extracted out into a separate function.

Reply:
Changed